Update priority feature sequence diagrams

g-lilian opened the pull request.

g-lilian committed:
Rename PPP and update AboutUs

g-lilian committed:
Add navigator to top of UG

g-lilian committed:
Add delete-plan and list-priority to UG

g-lilian committed:
Minor fixes to UG

g-lilian committed:
Update sequence diagrams

Updated setPriorityLevel sequence diagram and priority feature class diagram

Zhilin-Huang commented on docs/DeveloperGuide.md:
```diff
@@ -87,7 +87,7 @@ The following sequence diagram shows the relevant interactions behind `setPriori
 #### Study Plan Feature
 
-##### Proposed Implementation
+#### Proposed Implementation
```
Hmmm shouldn't we keep `Proposed Implementation` as a lower level header (with 5 #'s) than `Study Plan Feature` (with 4 #'s)?

g-lilian replied:
I changed it because in the actual website, the Proposed Implementation headers are of an even smaller font than the main text. Not sure if there's a better way to resolve this :O

Zhilin-Huang replied:
Maybe we could merge `Proposed Implementation` with `Study Plan Feature` and have `Study Plan Feature - Proposed Implementation` with 4 #'s? `Proposed Implementation` is the only subsection under `Study Plan Feature` anyway haha

g-lilian replied:
Updated all the headers according to your suggestion!
I also added the confirmation message to setPriorityLevel seq diag to keep it consistent with yours.

Zhilin-Huang replied:
> Updated all the headers according to your suggestion!
> I also added the confirmation message to setPriorityLevel seq diag to keep it consistent with yours.

Nice!

Zhilin-Huang commented on docs/images/setPriorityLevelSequenceDiagram.drawio:
```diff
@@ -1 +1 @@
-<mxfile host="app.diagrams.net" modified="2020-03-24T06:39:38.902Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36" etag="MSFUhgy4KfkcSFgPl4GM" version="12.9.0" type="device"><diagram id="o_KrSBOa8JoWa7Z2Ccje" name="Page-1">5Vxtb6M4EP41kXoftgKbl/CxzW5vT+qeetpd7fW+UXAT3xIcgdMm++vPBBPANoXkMKRJVSlhYgx4nhk/M2MzgbPl5vfEXy2+kBB ...
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2020-04-08T09:36:29.765Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36" etag="IsuZUEHFnnHOOmfLaUHO" version="12.9.0" type="device"><diagram id="o_KrSBOa8JoWa7Z2Ccje" name="Page-1">3VxRc5s4EP41mck9NAMICXhM3Ka9mfQmN22n13vDINtcseXBcmL3159khAEJB+wi4SEPrVmEAO1+u9+uJG7AZLn7mIXrxWcS4/T ...
```
Do you want to color the `Flashcard` class as well hahaha

g-lilian replied:
Coloured >_<

g-lilian committed:
Add colour to Flashcard

g-lilian committed:
Add colour to Flashcard again

g-lilian committed:
Add colour to Flashcard again

Oops, made a careless mistake just now

g-lilian committed:
Merge branch 'master' of https://github.com/AY1920S2-CS2113-T14-1/tp

g-lilian committed:
Update DG headers

Updated all feature headers to format ####XXX Feature - Proposed Implementation

g-lilian committed:
Add confirmation message to setPriorityLevel seq diag

Zhilin-Huang reviewed (approved):
LGTM!
